src/validation/ValidationContext.js

```diff
@@ -226,4 +244,15 @@ export default class ValidationContext {
   getArgument(): ?GraphQLArgument {
     return this._typeInfo.getArgument();
   }
+
+  // All OperationDefinitions, or FragmentDefinitions with variable definitions
+  isExecutableDefinitionWithVariables(
+    definition: ExecutableDefinitionNode,
+  ): boolean {
```

Comment:
This function doesn't use `this` it should be a standalone function, not a class method.

Reply:
Also maybe it worth to add `%checks` so Flow will have better type coverage.